mediapipe/tasks/python/test/vision/object_detector_test.py

```diff
@@ -136,6 +138,32 @@ def setUp(self):
         os.path.join(_TEST_DATA_DIR, _MODEL_FILE)
     )
 
+  def _assert_approximately_equals_bounding_box(self, bbox, expected_bbox):
+    self.assertEqual(bbox.width, expected_bbox.width)
+    self.assertEqual(bbox.height, expected_bbox.height)
```

Comment:
Not sure if we need to account for any numerical instability errors in width and height here too. PTAL.